### ISSUE TITLE
another case when russian text not matched

### ISSUE THREAD

**Issue by acidnik** [BurntSushi/ripgrep#264]
follow up to #251 
```
$ cat test.txt
слово другое
слова другие
слову другому
$ grep 'слов.*друг' test.txt
слово другое
слова другие
слову другому
$ rg 'слов.*друг' test.txt
$ rg --version
ripgrep 0.3.1
```

**Comment by BurntSushi** [comment]
This appears to be a duplicate of #251 and was therefore fixed in 0473df1ef5721143941fb7f883e22b17292b35bb. I can confirm it's working fine on current master:

```
$ cat test
слово другое
слова другие
слову другому
$ grep 'слов.*друг' test
слово другое
слова другие
слову другому
$ rg 'слов.*друг' test
1:слово другое
2:слова другие
3:слову другому
```

I'll try to get a new release out in the next few days.

**Comment by acidnik** [comment]
Sorry, I thougth 251 already released
